pyworkflow/mapper/sqlite.py

```diff
@@ -523,9 +522,9 @@ def _initialize(self):
         
     def __createTables(self):
         """Create required tables if don't exists"""
-        # Enable foreings keys
+        # Enable foreign keys
         self.setVersion(self.VERSION)
-        self._pragmas['foreing_keys'] = "ON"
+        self._pragmas['foreign_keys'] = "ON"
```

Comment:
I'm concern about this one
I can see that "foreing_keys" si wrong, and probably....not having any effect.
Now, with the correction, this will be properly activated, and may raise some constrains error?

To be honest, I don't think we re using foreign keys in scipion, do we? @delarosatrevin , @rmarabini

Reply:
I think that we are not using it, but not completely sure. Maybe to be on the safe side we can postpone this pragma activation until properly tested after the release.